src/main/java/net/snowflake/client/core/SFJsonResultSet.java

```diff
@@ -78,11 +83,32 @@ public Object getObject(int columnIndex) throws SFException {
       case Types.BOOLEAN:
         return getBoolean(columnIndex);
 
+      case Types.STRUCT:
+        if (Boolean.valueOf(System.getProperty("STRUCTURED_TYPE_ENABLED"))) {
```

Comment:
the constant `STRUCTURED_TYPE_ENABLED ` is used more than one - let's create real constant

Reply:
ok